Classes/Neos/Neos/Ui/Domain/Model/Changes/Property.php

```diff
@@ -137,7 +137,11 @@ public function apply()
             $value = $this->nodePropertyConversionService->convert(
                 $node->getNodeType(), $this->getPropertyName(), $this->getValue(), $node->getContext());
 
-            $node->setProperty($this->getPropertyName(), $value);
+            if ($this->getPropertyName() === '_hidden') {
```

Comment:
this should be generalized to using ObjectAccess::setProperty when the first character starts with "_"